web-app/src/pages/Dashboard/container.js

```diff
@@ -27,11 +28,15 @@ const DashboardContainer = Wrapped =>
     }
 
     componentDidMount() {
+      var self = this;
```

Comment:
What's this doing?

Reply:
Oh shit forgot to refactor this. 
I had a Promise in there which made 'this' refer to the Promise